includes/wikia/FullRequestLogger.class.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+class FullRequestLogger {
```

Comment:
Let's verify if this is still needed. In the era of global auth pages, `UserLoginSpecialController` is no longer used, and the Digital Ocean IPs may well be out of date. We might also have other sources for such access logs.

Reply:
Why do we even have hardcoded IPs there? It sounds wrong.

Reply:
It looks like it was introduced back in security incident days I think we should do at CDN level not inside application.

Reply:
Okay, I will review it and remove if no longer needed.